asm/istio/expansion/README.md

```diff
@@ -0,0 +1,20 @@
+# Anthos Service Mesh Expansion
+
+There are currently a few files in this directory that supports expansion to the
+mesh.
+
+- `gen-eastwest-gateway.sh` can be used to generate a custom `IstioOperator`
+configuration for the expansion gateway deployment. Please refer to the current
+Anthos Service Mesh [user guide](https://cloud.google.com/service-mesh/docs/gke-on-prem-install-multicluster-vmware)
+on how to use the script.
+
+- `vm-eastwest-gateway.yaml` is a generated `IstioOperator` configuration for
+the expansion gateway deployment to be used when adding VM workloads on the same
+network to the mesh. Using the configuration file, the expansion gateway will be
+installed when there is an existing control plane in the cluster.
```

Comment:
Make this active voice and clearer. "Don't use this configuration as an overlay when installing any other ASM components, this is only to add the expansion gateway to an existing installation."

Reply:
done.